lib/common.sh

```diff
@@ -93,11 +93,14 @@ FILESYSTEM=${FILESYSTEM:="/"}
 # BMO_RUN_LOCAL : run the Baremetal Operator locally (not in Kubernetes cluster)
 # CAPM3_RUN_LOCAL : run the Cluster API Provider Metal3 locally
 
-export CAPI_VERSION="${CAPI_VERSION:-"v1alpha4"}"
 export CAPM3_VERSION="${CAPM3_VERSION:-"v1alpha5"}"
-CAPM3_VERSION_LIST="v1alpha4 v1alpha5"
-if ! echo "${CAPM3_VERSION_LIST}" | grep -wq "${CAPM3_VERSION}"; then
-  echo "Invalid CAPM3 version : ${CAPM3_VERSION}. Not in : ${CAPM3_VERSION_LIST}"
+
+if [ "${CAPM3_VERSION}" == "v1alpha5" ]; then
+  export CAPI_VERSION="${CAPI_VERSION:-"v1alpha4"}"
+elif [ "${CAPM3_VERSION}" == "v1alpha4" ]; then
+  export CAPI_VERSION="${CAPI_VERSION:-"v1alpha3"}"
+else
+  echo "Invalid CAPM3 version : ${CAPM3_VERSION}. Not in : {v1alpha4, v1alpha5}"
```

Comment:
I did not meant the whole piece of code to be checked, but just restore `CAPM3_VERSION_LIST` as a separate var it used to be and use that as you still need it in current state of the patch, i.e:
```
CAPM3_VERSION_LIST="v1alpha4 v1alpha5"
if [ "${CAPM3_VERSION}" == "v1alpha5" ]; then
  export CAPI_VERSION="${CAPI_VERSION:-"v1alpha4"}"
elif [ "${CAPM3_VERSION}" == "v1alpha4" ]; then
  export CAPI_VERSION="${CAPI_VERSION:-"v1alpha3"}"
else
  echo "Invalid CAPM3 version : ${CAPM3_VERSION}. Not in : ${CAPM3_VERSION_LIST}"`

```